spec/models/spree/order_spec.rb

```diff
@@ -521,6 +521,7 @@
   end
 
   describe "applying enterprise fees" do
+    let(:subject) { create(:order) }
```

Comment:
This should rather be

```suggestion
    subject { create(:order) }
```

That's why it's called `subject` in the first place.